docs/content/users/quickstart.md

```diff
@@ -342,6 +342,76 @@ Here's a quickstart instructions for a number of different environments:
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
 
+=== "Craft CMS"
+
+    ## Craft CMS
+
+    [Craft CMS](https://craftcms.com) can be downloaded with [Composer](https://craftcms.com/docs/4.x/installation.html#downloading-with-composer), by cloning a [Craft starter project](https://github.com/craftcms/craft) or by [manually downloading](https://craftcms.com/docs/4.x/installation.html#downloading-an-archive-file-manually) a zipped archive. 
+    
+    === "New projects"
+    
+        ### Composer build
+        
+        Use this to create a new Craft CMS project from the official [Craft starter project](https://github.com/craftcms/craft) or a third-party starter project using Composer.
+        
+        ```bash
+        mkdir path/to/my-project
+        cd path/to/my-project
+        composer create craftcms/craft . --no-install --no-scripts
+        ddev config --project-type=craftcms
+        ddev start
+        ddev composer install
+        ddev craft setup
+        ddev launch
+        ```
+        
+        ### Git clone
+        
+        Use this to create a new Craft CMS project from the official [Craft starter project](https://github.com/craftcms/craft) or a third-party starter project using Git.
+        
+        ```bash
+        mkdir path/to/my-project
+        cd path/to/my-project
+        git clone https://github.com/craftcms/craft . && rm -rf .git*
+        ddev config --project-type=craftcms
+        ddev start
+        ddev composer install
+        ddev craft setup
+        ddev launch
+        ```
+
+        ### Manual download
+        
+        Use this to create a new Craft CMS project using a zipped archive.
+        
+        ```bash
+        mkdir path/to/my-project
+        cd path/to/my-project
+        curl -LO https://craftcms.com/latest-v4.zip
+        unzip latest-v4.zip && rm latest-v4.zip
+        ddev config --project-type=craftcms
+        ddev start
+        ddev composer install
+        ddev craft setup
+        ddev launch
+        ```
+
+    === Existing projects
+    
+        ### Git clone
+        
+        Use this to migrate an existing Craft CMS project from a git repository and import a database dump.
+        
+        ```bash
+        git clone https://github.com/example/example-site path/to/my-project
+        cd path/to/my-project
+        ddev config --project-type=craftcms
+        ddev start
+        ddev composer install
+        ddev import-db
```

Comment:
Although this works without flags, people seem to be confused by the behavior and an explicit example is probably better.
```suggestion
        ddev import-db --src=/path/to/db.sql.gz
```

Reply:
Seems reasonable